src/node-jspm-jasmine.js

```diff
@@ -88,26 +85,67 @@ export function runTests(opts, errCallback = function() {}) {
 				console.log(`Be sure to provide opts.coverage.files if using the JS API, or --coverage-files <pattern> if using the CLI. Also, be sure to put the pattern in quotes to avoid the glob being expanded before jspmjasmine executes.`);
 			}
 
-			SystemJS.instantiate = (load) => {
-				let normalizedAddress = getOSFilePath(load.address)
-				if (coverageFiles[normalizedAddress]) {
-					load.address = normalizedAddress;
-					coveredModules.push(load.address);
-					if ( load.metadata.sourceMap ) {
-						delete load.metadata.sourceMap.sourcesContent;
-						load.metadata.sourceMap.file = load.address + tranpiledFileSuffix
-						load.metadata.sourceMap.sources = load.metadata.sourceMap.sources.map(
-							filename => getOSFilePath(filename)
-						)
-						fs.writeFileSync(
-							load.address + tranpiledFileSuffix,
-							load.source  + '\n' + inlineSourceMap(load.metadata.sourceMap)
+			// create systemjs hook to allow Istanbul to instrument transpiled sources
+			const instrument = new Instrumenter();
+			const systemInstantiate = SystemJS.instantiate;
+			const tempDirectory = path.join(__dirname, '../tmp/');
```

Comment:
So when someone has installed node-jspm-jasmine, we put the temp directory into `node_modules/node-jspm-jasmine` (since this file is in `node_modules/node-jspm-jasmine/lib/node-jspm-jasmine.js`)? Just a question -- I think that that is fine